strings/tips/marine.txt

```diff
@@ -1,16 +1,26 @@
+Go to Valhalla to experiment with loadouts and weapons. You don't reset your respawn time in Valhalla!
+Fire disables pheromones, use this information to your advantage.
+Catch larval facehuggers by having an empty hand and throw mode on.
+Standing in a corner on harm intent stops you from behing shuffled, unless you're a marine and a xenomorph on help intent walks into you.
+Jumping will allow you to throw grenades farther, and over allies. Yeet those grenades to xenomorphs!
+Hunger will slow you down. A single protein bar will fill many, if not every, people. Take an extra protein bar too much and you become slower (ironic) and fat.
```

Comment:
will fill many, if not every, people

lol i get its funny but it was removed for a reason